apache2/apache2_config.c

```diff
@@ -30,6 +30,14 @@
     APLOG_USE_MODULE(security2);
 #endif
 
+// Returns the rule id if existing, otherwise the file name & line number
+static const char* id_log(msre_rule* rule) {
+	char* id = rule->actionset->id;
```

Comment:
This should be `const`:
```C
        const char* id = rule->actionset->id;
```

Reply:
It was static because it was duplicated in 2 files. I merged them